keadm/cmd/keadm/app/cmd/util/common_test.go

```diff
@@ -152,3 +153,44 @@ func TestPrivateDownloadServiceFile(t *testing.T) {
 		clean(testTmpDir)
 	})
 }
+
+func TestGetPackageManager(t *testing.T) {
+	if pm := GetPackageManager(); pm == "" {
+		t.Errorf("failed to get package manager")
+	}
+}
+
+func TestGetLatestVersion(t *testing.T) {
+	_, err := GetLatestVersion()
+	if err != nil {
+		t.Errorf("failed to query kubeedge version: %v", err)
+	}
+}
+
+func TestHasSystemd(t *testing.T) {
+	if !hasSystemd() {
+		t.Errorf("don't have systemd.")
```

Comment:
If you work in an environment without 'systemd', it may cause the unit test to fail all the time?